documentation/src/main/asciidoc/topics/contributing/ui.adoc

```diff
@@ -0,0 +1,29 @@
+[[design_guidelines]]
+= UI Writing Guidelines
+
+This section describes guidelines for working with text in {brandname} user
+interface (UI) elements and applies, but is not limited, to the following
+components:
+
+* {brandname} Console
+* {brandname} CLI
+* log messages
+* error messages
+
+Content in the {brandname} UI should adhere to the link:https://www.patternfly.org/v4/ux-writing/about[PatternFly content style guide].
```

Comment:
We call this the "PatternFly UX writing style guide." The name matches the nav item so that it's easier to find 👍